.github/test-spec.yml

```diff
@@ -81,7 +81,21 @@
   - "**/*"
 
 "CI-modemshell-test":
-  - "**/*"
+  - "include/net/**/*"
+  - "include/posix/**/*"
+  - "include/shell/**/*"
+  - "drivers/net/**/*"
+  - "drivers/serial/**/*"
+  - "drivers/wifi/**/*"
+  - "subsys/shell/**/*"
+  - "subsys/net/**/*"
+  - "subsys/settings/**/*"
 
 "CI-positioning-test":
-  - "**/*"
+  - "include/net/**/*"
+  - "include/posix/**/*"
+  - "include/shell/**/*"
```

Comment:
`include/shell` is probably not needed, because these tests do not use the shell.